packages/protocol/scripts/bash/release-on-devchain.sh

```diff
@@ -63,7 +63,8 @@ yarn ts-node scripts/check-backward.ts sem_check --old_contracts $BUILD_DIR/cont
 
 # From make-release.sh
 echo "- Deploy release of current branch"
-yarn truffle exec --network development ./scripts/truffle/make-release.js --build_directory build/ --report report.json --proposal proposal.json --initialize_data example-initialize-data.json
+INITIALIZATION_FILE=`ls releaseData/initializationData/* | tail -n 1`
```

Comment:
This could be very fragile with how we're numbering the release jsons (wouldn't the file for release 10 be hidden by release 9 due to lexicographic ordering?)

Reply:
a744d1a96